cli/daemon/run/watch.go

```diff
@@ -37,12 +39,17 @@ func (mgr *Manager) watch(run *Run) error {
 				time.Sleep(100 * time.Millisecond)
 				mgr.RunStdout(run, []byte("Changes detected, recompiling...\n"))
 				if err := run.Reload(); err != nil {
-					errStr := err.Error()
-					if !strings.HasSuffix(errStr, "\n") {
-						errStr += "\n"
+					if errList := errlist.Convert(err); errList != nil {
+						mgr.RunError(run, errList)
+					} else {
+						errStr := err.Error()
+						if !strings.HasSuffix(errStr, "\n") {
+							errStr += "\n"
+						}
+						mgr.RunStderr(run, []byte(errStr))
 					}
-					mgr.RunStderr(run, []byte(errStr))
 				} else {
+					mgr.RunError(run, nil)
```

Comment:
Is this needed? That's very unintuitive, if so

Reply:
For the bubbletea UI I want to add yes, I'll remove it for now though